x-pack/plugins/security_solution/public/management/cypress/screens/alerts.ts

```diff
@@ -12,8 +12,10 @@ export const navigateToAlertsList = (urlQueryParams: string = '') => {
   loadPage(`${APP_ALERTS_PATH}${urlQueryParams ? `?${urlQueryParams}` : ''}`);
 };
 
-export const clickAlertListRefreshButton = (): Cypress.Chainable => {
-  cy.getByTestSubj('querySubmitButton').first().click();
+const clickAlertListRefreshButton = (): Cypress.Chainable => {
+  cy.getByTestSubj('filters-global-container').within(() => {
```

Comment:
can we do it differently? meaning, without using `within` ? Can you check if this would work?

   ```
   cy.get(
      '[data-test-subj="filters-global-container"] [data-test-subj="querySubmitButton"]'
    ).click();

Reply:
Changed.